README.md

```diff
@@ -16,28 +16,55 @@ To guide development we [have documented the user needs](docs/NEEDS.md) and [ass
 
 ## Usage
 
+To run a GOV.UK application, `govuk-docker build` it first (you only need to do this once per application):
+
 ```sh
-# Run a Rake task on Whitehall
-whitehall$ govuk-docker run bundle exec rake -T
+# This assumes you're in the ~/govuk/govuk-docker directory.
```

Comment:
The following command can actually be run from any directory - and I think in general I think we would expect most people to run it (and the other commands) from the directory of the app itself to avoid having to pass `--service` around. I think it's still a good idea to publicise the `--service` option, but my making it the first thing people see, they may assume you have to do it this way and not read on.